content/en/news/releases/1.22.x/announcing-1.22/change-notes/index.md

```diff
@@ -0,0 +1,246 @@
+
+- **Added** support for path templating in AuthorizationPolicy, See Envoy URI template [docs](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/path/match/uri_template/v3/uri_template_match.proto).  ([Issue #16585](https://github.com/istio/istio/issues/16585))
+
+- **Added** support for customizing timeout when resolving `jwksUri`
+  ([Issue #47328](https://github.com/istio/istio/issues/47328))
+
+- **Added** support for Istio CA to handle node authorization for CSRs with impersonate identity from remote clusters.
+This could help Istio CA to authenticate ztunnel in remote clusters in an external control plane scenario.  ([Issue #47489](https://github.com/istio/istio/issues/47489))
+
+- **Added** An environment variable `METRICS_LOCALHOST_ACCESS_ONLY` for disabling metrics endpoint from outside of the pod, to allow only localhost access. User can use set this with command
+`--set values.pilot.env.METRICS_LOCALHOST_ACCESS_ONLY=true` for Control plane and `--set meshConfig.defaultConfig.proxyMetadata.METRICS_LOCALHOST_ACCESS_ONLY=true` for proxy while istioctl installation.
+
+- **Added** Certificate Revocation List(CRL) support for peer certificate validation based on file paths specified in `ClientTLSSettings` in destination rule for Sidecars and in `ServerTLSSettings` in Gateway for Gateways.
+
+- **Fixed** list matching for the audience claims in JWT tokens.
+  ([Issue #49913](https://github.com/istio/istio/issues/49913))
+
+- **Removed** the `first-party-jwt` legacy option for `values.global.jwtPolicy`. Support for the more secure `third-party-jwt`
+has been default for many years and is supported in all Kubernetes platforms.
+
+## Telemetry
+
+- **Improved** JSON access logs to emit keys in a stable ordering.
+
+- **Added** option to export OpenTelemetry traces via HTTP
+ ([reference]( https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ExtensionProvider-OpenTelemetryTracingProvider)) ([Issue #47835](https://github.com/istio/istio/issues/47835))
+
+- **Enabled** configuring Dynatrace Sampler for the `OpenTelemetryTracingProvider` in `MeshConfig`.
+  ([Issue #50001](https://github.com/istio/istio/issues/50001))
+
+- **Enabled** configuring Resource Detectors for the `OpenTelemetryTracingProvider` in `MeshConfig`.
+  ([Issue #48885](https://github.com/istio/istio/issues/48885))
+
+- **Fixed** an issue that `TraceId` is not propagated when using OpenTelemetry access logger.
+  ([Issue #49911](https://github.com/istio/istio/issues/49911))
+
+- **Removed** default tracing configuration that enables tracing to `zipkin.istio-system.svc`. See upgrade notes for more information.
+
+## Extensibility
+
+- **Improved** Use tag-stripped URL + checksum as a Wasm module cache key, and the tagged URL is separately cached.
+This may increase the chance of cache hit (e.g., trying to find the same image with both of the tagged and digest URLs.)
+In addition, this will be a base to implement `ImagePullPolicy`.
```

Comment:
```suggestion
- **Improved** using the tag-stripped URL and checksum as a Wasm module cache key, where the tagged URL is separately cached.
  This may increase the chance of cache hits (e.g., trying to find the same image with both of the tagged and digest URLs.)
  In addition, this will be a base to implement `ImagePullPolicy`.
```